packages/client/hmi-client/src/components/funman/tera-funman-output.vue

```diff
@@ -20,20 +20,43 @@
 				{{ title }}
 			</header>
 		</div>
+
 		<div v-for="(bound, parameter) in lastTrueBox?.bounds" :key="parameter">
-			<div class="variables-row" v-if="parameterOptions.includes(parameter.toString())">
-				<RadioButton v-model="selectedParam" :value="parameter.toString()" />
-				<div>{{ parameter.toString() }}</div>
-				<InputText v-model="bound.lb" />
-				<InputText v-model="bound.ub" />
+			<div class="variables-row" v-if="parameterOptions.includes(parameter)">
+				<RadioButton v-model="selectedParam" :value="parameter" />
+				<div>{{ parameter }}</div>
+				<InputNumber
+					mode="decimal"
+					:min-fraction-digits="1"
+					class="p-inputtext-sm"
+					v-model="bound.lb"
+				/>
+				<InputNumber
+					mode="decimal"
+					:min-fraction-digits="1"
+					class="p-inputtext-sm"
+					v-model="bound.ub"
+				/>
 				<TeraFunmanBoundaryChart
 					:processed-data="processedData as FunmanProcessedData"
-					:param1="parameter.toString()"
-					:param2="selectedParam"
+					:param1="selectedParam"
+					:param2="parameter"
 					:timestep="timestep"
+					@click="selectedParam2 = parameter"
 				/>
 			</div>
 		</div>
+
+		<!-- Larger version of boundary chart: TODO -->
+		<div v-if="selectedParam2">
+			<TeraFunmanBoundaryChart
+				:processed-data="processedData as FunmanProcessedData"
+				:param1="selectedParam"
+				:param2="selectedParam2"
+				:options="{ width: 350, height: 250 }"
+				:timestep="timestep"
+			/>
+		</div>
```

Comment:
this is not ready, I will fix this section in the next PR - it is meant to be a drilldown-version of the boundary-chart.